nuclear_simulator/data_gen/config_engine/templates/nuclear_plant_comprehensive_config.yaml

```
@@ -0,0 +1,1190 @@
+          threshold: 0.25              # MPa suction pressure (FIXED: above 0.2 MPa protection trip)
+          comparison: "less_than"
+          action: "npsh_analysis"
+          cooldown_hours: 168.0        # Weekly cooldown
+          priority: "HIGH"
+        cavitation_inception:
+          threshold: 0.86              # 86% of design flow for cavitation
+          comparison: "less_than"
+          action: "cavitation_analysis"
+          cooldown_hours: 168.0        # Weekly cooldown
+          priority: "HIGH"
+        cavitation_intensity:
```

YAMLlint (1.37.1)

[error] duplication of key "cavitation_intensity" in mapping

(key-duplicates)